vng_api_common/tests/auth.py

```diff
@@ -59,7 +59,6 @@ def generate_jwt_auth(
         "user_representation": user_representation,
     }
     encoded = jwt.encode(payload, secret, algorithm="HS256")
-    encoded = encoded.decode("ascii")
```

Comment:
@sergei-maertens Is it okay to assume that these tests will be run using PyJWT >= 2.0.0, or should it still be compatible with PyJWT<2.0.0?

Reply:
you can assume tests run with old PyJWT version, but the existing projects may have PyJWT < 2.0 so the old variant should still work :-)